openfisca_us/variables/demographic/person.py

```diff
@@ -45,6 +45,55 @@ def formula(person, period, parameters):
         )
 
 
+class is_ccdf_home_based(Variable):
+    value_type = bool
+    default_value = False
+    entity = Person
+    label = u"Whether CCDF care is home-based versus center-based"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        return (
+            person("provider_type_group", period) != ProviderTypeGroup.DCC_SACC
+        )
+
+
+class CCDFAgeGroup(Enum):
+    INFANT = "Infant"
+    TODDLER = "Toddler"
+    PRESCHOOLER = "Preschooler"
+    SCHOOL_AGE = "School age"
+
+
+class ccdf_age_group(Variable):
+    value_type = Enum
+    possible_values = CCDFAgeGroup
+    default_value = CCDFAgeGroup.INFANT
```

Comment:
@nikhilwoodruff should we have default values for variables that descend from other variables with their own default values? e.g. this is based on age (default 0) and home-based (default False), so it would default to infant anyway.

Reply:
Let's actually deal with this question separately. Merging.